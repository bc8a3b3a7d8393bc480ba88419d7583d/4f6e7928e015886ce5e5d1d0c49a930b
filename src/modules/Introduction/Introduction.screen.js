import React, { Component } from 'react';

import IntroductionView from './components/Introduction.view';

export class IntroductionScreen extends Component {
  render() {
    return <IntroductionView />;
  }
}

export default IntroductionScreen;
