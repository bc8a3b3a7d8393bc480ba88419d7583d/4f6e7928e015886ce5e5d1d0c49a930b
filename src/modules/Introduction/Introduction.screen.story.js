import React from 'react';

// eslint-disable-next-line import/no-extraneous-dependencies
import { storiesOf } from '@storybook/react-native';

import Component from './Introduction.screen';

storiesOf('Modules/Introduction - Introduction Screen', module).add(
  'default',
  () => (
    <Component
      navigation={{
        navigate: () => console.log('navigate'),
      }}
    />
  ),
);
