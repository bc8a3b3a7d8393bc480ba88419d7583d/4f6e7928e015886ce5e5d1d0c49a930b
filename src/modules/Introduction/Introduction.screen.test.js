import { testSnapshots } from '../../utils/test.util';

import { IntroductionScreen as Component } from './Introduction.screen';

describe('Modules/Landing - Introduction screen', () => {
  testSnapshots(Component, [
    {
      description: 'basic render',
    },
  ]);
});
