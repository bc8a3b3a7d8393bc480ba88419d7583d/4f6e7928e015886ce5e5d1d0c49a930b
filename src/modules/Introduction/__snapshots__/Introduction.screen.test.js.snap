// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modules/Landing - Introduction screen snapshots basic render 1`] = `<IntroductionView />`;
